ruby.js

```diff
@@ -0,0 +1,42 @@
+const io = require("@actions/io")
+const { readFile } = require("fs")
+const { promisify } = require("util")
+const { easyExec } = require('./utils')
+
+const readFileAsync = promisify(readFile)
+
+gemVersion = null
```

Comment:
What would you think about assigning this with a `let`? I'm not sure if declaring the variable in this way will cause any actual issues (in some contexts, it'd be an implicit global, but not sure about this particular usage in nodejs), but maybe using `let` would be a bit more clear as to how we expect this to be used?